app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileRepo.kt

```diff
@@ -75,7 +80,7 @@ class PinnedTileRepo(private val applicationContext: Application) {
     @UiThread
     fun removePinnedTile(url: String): String? {
         val tileToRemove = _pinnedTiles.value?.remove(url) ?: return null
-        _pinnedTiles.value = _pinnedTiles.value
+        _pinnedTiles.onNext(_pinnedTiles.value!!)
```

Comment:
I guess this was a problem with LD too, and just wasn't caught by the compiler, but I'm not super happy about `!!`'ing this.  `BehaviorSubject#getValue` is nullable because it can be accessed before any values have been pushed.  We're pushing values on init, so this won't crash at the moment, but I'm still wary of it.

This could be refactored to follow a more reactive approach, but I don't think it's worth the effort right now.  So no change needed here, but bear in mind this kind of danger in the future.